Sources/Core/AWSSDKEventStreamsAuth/Context+AWSEventStreamsAuth.swift

```diff
@@ -11,25 +11,26 @@ import SmithyEventStreamsAPI
 import SmithyEventStreams
 import SmithyEventStreamsAuthAPI
 
-extension Context {
+/// Setups context with encoder, decoder and signer for bidirectional streaming
+/// and sets the bidirectional streaming flag
+/// - Parameter context: The context to be configured for bidirectional streaming
+public func setupBidirectionalStreaming(context: Context) {
```

Comment:
Changed this function to a static function from an extension on Context so it can be individually imported.